packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1256,7 +1314,6 @@ class StackViewport extends Viewport implements IStackViewport {
     this.currentImageIdIndex = currentImageIdIndex;
     this.targetImageIdIndex = currentImageIdIndex;
     this.stackInvalidated = true;
-    this.rotationCache = 0;
```

Comment:
_resetProperties on line 1321 takes care of this since rotation is now solely fetched from the camera.